pkg/cmd/server/bootstrappolicy/securitycontextconstraints_test.go

```diff
@@ -2,20 +2,22 @@ package bootstrappolicy
 
 import (
 	"reflect"
+	"sort"
 	"testing"
 
 	"k8s.io/apiserver/pkg/authentication/serviceaccount"
 
 	securityapi "github.com/openshift/origin/pkg/security/apis/security"
+	scc "github.com/openshift/origin/pkg/security/securitycontextconstraints"
 )
 
 func TestBootstrappedConstraints(t *testing.T) {
 	expectedConstraints := []string{
```

Comment:
Consider to rename `expectedConstraints` to `expectedConstraintNames` to better express it's content.